src/me/StevenLawson/TotalFreedomMod/Commands/Command_nightvision.java

```diff
@@ -0,0 +1,33 @@
+package me.StevenLawson.TotalFreedomMod.Commands;
+
+import net.md_5.bungee.api.ChatColor;
```

Comment:
Hmm Interesting. I hadnt noticed that before. But yeah, really it should be the bukkit one as thats the API we are using, not Bungee ;)